templates/zerver/help/unread-counter-and-pointer.md

```diff
@@ -0,0 +1,75 @@
+# Navigation and unread counts
+
+## Unread messages
+
+Zulip is carefully designed to automatically track which messages
+you've read to produce the ideal reading experience, where you can
+always start reading where you left off:
+
+* Messages are automatically marked as read only when you're likely to
+have actually read them.
+
+* When you open a view in Zulip, it takes you to the first unread
+message in that view (if any).
+
+These goals are achieved through the following behaviors:
+
+* Essentially, Zulip will consider a message read when the message is
+selected (the blue cursor box passes over it). Any message which is
+selected or above a message which is selected will be marked as read.
+
+* Whenever you're in a view where the whitespace at the bottom of that
+view is visible, Zulip marks all the messages in that view as read.
+
+* If you're in a longer view where the bottom whitespace isn't
```

Comment:
This point talks about a lot of information and I think it'd be better to split it up into a series of subpoints to the previous point since it's an elaboration of it. For example:
```md
* Whenever you're in a view where the whitespace at the bottom of that view is visible...
    * If you're in a longer view where the bottom whitespace isn't visible...
    * If you're navigating with the keyboard, a message is marked as read...
    * If you're navigating with the mouse, Zulip automatically advances the blue cursor box...
```